pkgs/applications/misc/solaar/default.nix

```diff
@@ -1,40 +1,50 @@
 { fetchFromGitHub, lib, gobject-introspection, gtk3, python3Packages }:
-# Although we copy in the udev rules here, you probably just want to use logitech-udev-rules instead of
-# adding this to services.udev.packages on NixOS
+
+# Although we copy in the udev rules here, you probably just want to use
+# logitech-udev-rules instead of adding this to services.udev.packages on NixOS
```

Comment:
sorry, that was not my point. My point was that if somebody installs the package and it doesn't work, the description says what steps need to be done to enable udev rules (it can be to enable hardware.logitech, sure)

Reply:
How about now?

Reply:
looks cool, thank you for taking that into account